src/EditorFeatures/CSharp/Highlighting/KeywordHighlighters/DiscardParameterHighlighter.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.ComponentModel.Composition;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Editor.Implementation.Highlighting;
+using Microsoft.CodeAnalysis.Text;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.Highlighting.KeywordHighlighters
+{
+    [ExportHighlighter(LanguageNames.CSharp)]
+    internal class DiscardParameterHighlighter : AbstractKeywordHighlighter
```

Comment:
ther's a different baseclass you can subclass right?  where you can specify through a type argument that you are constrained to ArgumentSyntax right?

Reply:
yes, it's `AbstractKeywordHighlighter<ArgumentSyntax>`.